management-cluster-access/README.md

```diff
@@ -0,0 +1,95 @@
+For Giant Swarm staff this is done via `dex` using github as the _only_ identity provider. 
+
+Our vintage product also offers the possibility to create a client certificate to access the management cluster.
+This means that in the event of github or `dex` being unavailable, we do not get locked out of the management cluster.
+
+For pure CAPI installations on the other hand, no other authentication method is available at this point.
+This poses a real risk to get locked out of the management cluster in case of problems with OIDC.
+We also rely on the github api being available as an external dependency we can not control.
+
+Furthermore, loss of access to the management clusters implies loss of access to all workload clusters that do not have OIDC access for giant swarm staff set up.
+This is due to the fact that in order to create a client certificate for these workload clusters, we need to access the management cluster first.
+At this moment this likely applies to all workload clusters on pure CAPI installations.
+
+However, even if OIDC access was set up, it would still leave github as a single point of failure.
+
+Let's ensure that management cluster access is highly available for operations through GS staff in the future.
+
+## References
+
+- [PKI epic](https://github.com/giantswarm/giantswarm/issues/15981)
+- [Issue regarding client cert creation for CAPO ](https://github.com/giantswarm/giantswarm/issues/21740)
+- [Story for backup IDP in dex](https://github.com/giantswarm/roadmap/issues/603)
+- [Proposal to use Azure AD for GS staff](https://github.com/giantswarm/giantswarm/issues/21627)
+- 
+
+## Possible solutions
+
+### Client certificates
+
+In the vintage product, client certificates can be created from outside the cluster using `vault` for signing.
+At the moment this is not the case for the CAPI product.
+
+- This method of authentication is completely separate from SSO and does not have shared dependencies.
+- As of now, client certificate creation is harder to monitor and control than SSO.
+- Client certificates should be short lived and therefore would need to be recreated by users frequently.
+- A long living client-certificate granting admin access is a security risk since permissions can not be revoked.
+- We do not want this to be the primary authentication method for human users. (That should remain SSO)
+- If we want to support this, we need to decide whether we want to use vault or something else.
+
+### Storing kubeconfig in lastpass
+
+In the CAPI product, a valid kubeconfig is stored in lastpass as emergency fallback. This is part of the bootstrap process.
+
+- This method of authentication is completely separate from SSO and does not have shared dependencies.
+- It is not integrated in our tooling and we need to pull the kubeconfig using lastpass cli.
+- We should revisit how this works. What type of authorization is used here? Is access revokable? Is this rotated?
```

Comment:
No, and only way to rotate it would be rotating API server CA and all certs, it's not a great thing security wise